config_serviceflags.go

```diff
@@ -8,19 +8,18 @@ import (
 
 // ServiceFlags holds the state of each of the runner/API processes
 type ServiceFlags struct {
-	TaskDispatchDisabled         bool `bson:"task_dispatch_disabled" json:"task_dispatch_disabled"`
-	HostinitDisabled             bool `bson:"hostinit_disabled" json:"hostinit_disabled"`
-	MonitorDisabled              bool `bson:"monitor_disabled" json:"monitor_disabled"`
-	AlertsDisabled               bool `bson:"alerts_disabled" json:"alerts_disabled"`
-	TaskrunnerDisabled           bool `bson:"taskrunner_disabled" json:"taskrunner_disabled"`
```

Comment:
We should remove this field from the production and staging databases after deploying this change.